searx/settings.yml

```diff
@@ -245,7 +245,7 @@ plugins:
     active: false
 
   searx.plugins.tracker_url_remover.SXNGPlugin:
-    active: false
```

Comment:
I think this shouldn't be by default because this may have potential side effects that we do not know.

And we do not maintain the list in "CLEAR_LIST_URL".

It's like hosting a pihole or adguard home, you start to add multiple sources but then some websites starts to break or "half break".

The user should be aware of such feature by turning it on by themselves.